<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormInteraction</class>
 <widget class="QWidget" name="FormInteraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter intercation et taches</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Titre de l'interaction :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titreEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="newIButton">
       <property name="text">
        <string>Nouvelle Interaction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveIButton">
       <property name="text">
        <string>Enregistrer Interaction</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="AnnulerButton">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Contenu :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="contenu"/>
     </item>
     <item>
      <widget class="QLabel" name="m2Label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AjoutButton">
       <property name="text">
        <string>Ajouter Tache</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m1Label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
